scripts/install-cni.sh

```diff
@@ -183,6 +183,8 @@ else
   echo "Creating CNI spec..."
 fi
 
-cat >${output_file} <<EOF
+cat > "${output_file}.tmp" <<EOF
```

Comment:
You may want to use `mktemp` to create the tempfile, which should be available on Alpine.
Also, it would be good form to delete the tempfile unconditionally with a `trap`, e.g.
```
trap "rm -f $tempfile" EXIT
```

Reply:
LGTM, thanks!